Jenkinsfile

```diff
@@ -48,6 +49,11 @@ pipeline {
                 sh "python add_redirects.py $major_version $minor_version stan-users-guide"
             }
         }
+        stage("Link docs to latest") {
+            steps{
+                sh "add_links.sh $major_version $minor_version $patch_version"
```

Comment:
we need to add links to the previous version, not the version we're building the release for -
the old hack script had 2_20 hardcoded - that's because at the point I wrote it we were at 2.21